examples/build.gradle

```diff
@@ -0,0 +1,13 @@
+dependencies {
+    compile project(path: ':client')
+    compile project(path: ':server')
+    //Use jdk14 bindings for the sample application
```

Comment:
`examples` module is no longer a part of `core-java`. We will be updating it to v0.6.0 right after this PR is merged to master.